app/src/test/java/com/github/multimatum_team/multimatum/DeadlineAdapterTest.kt

```diff
@@ -0,0 +1,124 @@
+package com.github.multimatum_team.multimatum
+
+import android.app.Application
+import android.graphics.Color
+import android.graphics.Typeface
+import android.view.View
+import android.widget.ListView
+import android.widget.TextView
+import androidx.test.core.app.ApplicationProvider
+import androidx.test.ext.junit.runners.AndroidJUnit4
+import com.github.multimatum_team.multimatum.model.Deadline
+import com.github.multimatum_team.multimatum.model.DeadlineAdapter
+import com.github.multimatum_team.multimatum.model.DeadlineState
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import java.time.LocalDate
+
+@RunWith(AndroidJUnit4::class)
+class DeadlineAdapterTest {
+
+    private lateinit var adapter: DeadlineAdapter
+    private var context: Application? = null
+    private lateinit var list: List<Deadline>
+
+    @Before
+    @Throws(Exception::class)
+    fun setUp() {
+        list = listOf(Deadline("Number 1",DeadlineState.TODO, LocalDate.now().plusDays(1)),
+            Deadline("Number 2",DeadlineState.TODO, LocalDate.now().plusDays(7)),
+            Deadline("Number 3",DeadlineState.DONE, LocalDate.of(2022, 3,30)),
+            Deadline("Number 4",DeadlineState.TODO, LocalDate.of(2022, 3,1)))
+        context = ApplicationProvider.getApplicationContext()
+        adapter = DeadlineAdapter(context!!, list)
+    }
+
+
+    @Test
+    fun testGetCount(){
```

Comment:
Nitpick again: you could use Kotlin's ``fun `Explain the test scenario in plain english here`()`` syntax so that all of our tests are consistent.